tools/iso_countries.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3
+
+# Generates iso_countries.*.yaml files from WHO official list of countries:
+# "Detailed_Boundary_ADM0 (24).csv" (stored in Google Drive)
+
+import csv
+import sys
+import unicodedata
+
+iso_2_code = 'ISO_2_CODE'
+fields = {
+    'en': 'ADM0_VIZ_NAME',
+    'ar': 'ARABIC',
+    'es': 'SPANISH',
+    'fr': 'FRENCH',
+    'ru': 'RUSSIAN',
+    'ch': 'CHINESE'}
```

Comment:
"The standard locale for simplified Chinese is zh_CN . The standard locale for traditional Chinese is zh_TW". You'd need to decide which form of chinese you're using too.

Reply:
Good point. This must be simplified Chinese and `zh_CN` - I've changed it too that. I tried it on several detector sites and they confirmed simplified Chinese.